### DISCUSSION TITLE
Is it unreasonable to add folders as collections in the data model?

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#8922] [Archive]
It will appear in the permissions, making permissions settings very confusing.

**Comment by rijkvanzanten** [comment]
Heya! 

This sounds like a (potential) bug. The folder-collections shouldn't show up under permissions settings 👍🏻 

Mind [opening a new issue](https://github.com/directus/directus/issues/new)? Thanks!

**Reply by cdwmhcc** [reply]
https://github.com/directus/directus/issues/8935
